app/locales/en.json

```diff
@@ -809,10 +809,10 @@
   "powerdown_jsx": {
     "power_down": "Power Down",
     "power_down_amount": "Power Down Amount",
-    "select_powerdown_amt": "Please select and amount of %(VESTING_TOKEN) to power down.",
+    "select_powerdown_amt_VESTING_TOKEN": "Please select and amount of %(VESTING_TOKEN)s to power down.",
```

Comment:
why did the token name change from `select_powerdown_amt` to `select_powerdown_amt_VESTING_TOKEN`?